api/api.go

```diff
@@ -227,13 +230,16 @@ type FederationResponse struct {
 
 // caHandler is the type used to implement the different CA HTTP endpoints.
 type caHandler struct {
-	Authority Authority
+	Authority   Authority
+	acmeHandler *acmeHandler
 }
 
 // New creates a new RouterHandler with the CA endpoints.
 func New(authority Authority) RouterHandler {
 	return &caHandler{
 		Authority: authority,
+		acmeHandler: newACMEHandler(authority, authority.GetDatabase().(nosql.DB),
+			"ca.smallstep.com:8080", "acme", nil),
```

Comment:
I suppose this is just for testing right?

Reply:
yea